microsoft/testsuites/core/hv_module.py

```diff
@@ -222,7 +222,9 @@ def verify_hyperv_modules(self, log: Logger, environment: Environment) -> None:
             min_core_count=4,
         ),
     )
-    def reload_hyperv_modules(self, case_name: str, log: Logger, node: Node) -> None:
+    def stress_reload_hyperv_modules(
```

Comment:
It shouldn't be called stress. It's a P1 test case, and not slow.

Reply:
updated